.github/workflows/test-pull-request.yml

```
@@ -209,7 +159,7 @@ jobs:
   check-workflow-status:
     name: Check pull request workflow status job
     runs-on: ubuntu-latest
-    needs: [ validate, changes, compile-test, template-compile-test ]
+    needs: [ validate, changes, template-compile-test ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)